src/omv/ov5640.c

```diff
@@ -889,7 +897,7 @@ int ov5640_init(sensor_t *sensor)
     sensor->set_lens_correction = set_lens_correction;
 
     // Set sensor flags
-    SENSOR_HW_FLAGS_SET(sensor, SENSOR_HW_FLAGS_VSYNC, 0);
+    SENSOR_HW_FLAGS_SET(sensor, SENSOR_HW_FLAGS_VSYNC, 1);
```

Comment:
Are you sure about this ? If VSYNC was wrong, it shouldn't have worked... @kwagyeman

Reply:
I changed a register that was inverting the vsync signal too.